front/app/api/project_folders/__mocks__/useAddProjectFolder.ts

```diff
@@ -0,0 +1,77 @@
+export const projectFolderData = {
```

Comment:
I think this is a mock for `useProjectFolder` - the name of the file should reflect that.